src/plone/app/mosaic/transform.py

```diff
@@ -152,10 +152,18 @@ def transformIterable(self, result, encoding):
             ])
 
         # Get contentLayout body class
-        if 'template-layout' in body_classes:
+        if (
+            'template-layout' in body_classes or
```

Comment:
@thet Good point :/ That's probably because at some point the view was named @@layout and not @@layout_view. You could still do that change for 2.0. Since you have fresh touch on that module, would you be able to do that?

Although, if it was not to stay consistent with plone.app.layout classes, template-layout would look nicer ;)